app/services/host_inventory_api.rb

```diff
@@ -6,16 +6,19 @@
 # Interact with the Insights Host Inventory. Usually HTTP calls
 # are all that's needed.
 class HostInventoryAPI
-  def initialize(host, account, url, b64_identity)
-    @host = host
+  def initialize(id, hostname, account, url, b64_identity)
+    @id = id
+    @hostname = hostname
     @url = "#{URI.parse(url)}#{ENV['PATH_PREFIX']}/inventory/v1/hosts"
     @account = account
     @b64_identity = b64_identity
   end
 
-  def host_already_in_inventory
+  def host_already_in_inventory(hostname_or_id)
     response = Platform.connection.get(
-      @url, {}, 'X_RH_IDENTITY' => @b64_identity
+      @url,
+      hostname_or_id: hostname_or_id,
```

Comment:
Nice, I thought it was a mistake but indeed `hostname_or_id=d5bc2459-21ce-4d11-bc0b-03ea7513dfa6` works as a parameter to the URL...